helm-charts/splunk-otel-collector/values.yaml

```
@@ -185,6 +185,15 @@ splunkObservability:
 
 logsEngine: otel
 
+################################################################################
+# Migrating Fluentd's position file to Otel checkpoints
+# Aimed for users who have been using fluentd as a logging agent but wants to
+# use OpenTelemetry Collector.
+# When set to true, an initContainer is added to the daemonset and runs the migration
+################################################################################
+
+migrateCheckpoint: false
```

Maybe pick a more descriptive name given that it's on the root level?
```suggestion
migrateLogsCheckpoints: false
```